../vector_tests/erase(range).cpp:24:26: error: no member named 'erase' in 'ft::vector<int, std::__1::allocator<int> >'
    v.push_back(*(vector.erase(vector.begin() + 8 * _ratio, vector.end() - 1500 * _ratio)));
                  ~~~~~~ ^
../vector_tests/erase(range).cpp:34:66: note: in instantiation of function template specialization 'erase_test_2<int>' requested here
    exit(run_vector_unit_test<int>("erase(range)", erase_test_2, erase_test_2));
                                                                 ^
1 error generated.
